### DISCUSSION TITLE
Cannot create a basic Uno solution

### DISCUSSION THREAD

**Discussion by mrpmorris** [unoplatform/uno#16927] [General]
I have been following [these instructions](https://platform.uno/docs/articles/get-started-vs-2022.html?tabs=ubuntu1804) .

I installed and ran uno-check, and it gave the following output. It seems I have problems, but it doesn't tell me what they are...
![IMG_20240529_170014__01](https://github.com/unoplatform/uno/assets/3111981/6e49df02-2582-498f-a9df-729f74b9bd8e)

If I try to ignore what looks like a weird blip, I discover the creation of a solution doesn't work.
![image](https://github.com/unoplatform/uno/assets/3111981/91c70c9f-3a5f-4161-b2b7-8a9ee7709f33)

Can anyone tell me what to do?

Thanks

**Comment by agneszitte** [comment]
cc @jeromelaban, @dansiegel

**Reply by sasakrsmanovic** [reply]
Thanks for moving it from twitter to here @mrpmorris . This way it is more discoverable by anyone else with similar problem!

**Comment by francoistanguay** [comment]
Can this help? https://platform.uno/docs/articles/common-issues-all-ides.html#could-not-resolve-sdk-unosdk

If you indeed had mappings in there and clearing some helped, would be interesting to know if you were aware you had mappings in there, the reason, ...

**Reply by francoistanguay** [reply]
Looks like other people had same issue and fixed it: https://github.com/unoplatform/uno/discussions/16758#discussioncomment-9537824

We'd love to better understand what's going on and provide the VS team with more info as needed.

**Comment by mrpmorris** [comment]
This seems to have solved the problem, thank you!

I have endless problems with command-line tools not working simply because VS has added a nuget package source and holds auth info that command-line utilities don't have access to.

If you are using `dotnet` commands I believe you can add the `--ignore-failed-sources` to skip these problems.

**Reply by agneszitte** [reply]
cc @jeromelaban for the details
